oauthenticator/openshift.py

```diff
@@ -27,33 +31,65 @@ def _username_claim_default(self):
         os.environ.get('OPENSHIFT_URL')
         or 'https://openshift.default.svc.cluster.local',
         config=True,
+        help="""
+        Used to determine the default values for `openshift_auth_api_url` and
+        `openshift_rest_api_url`.
+        """,
     )
 
-    validate_cert = Bool(
-        True, config=True, help="Set to False to disable certificate validation"
+    allowed_groups = Set(
+        config=True,
+        help="""
+        Allow members of selected OpenShift groups to sign in.
+        """,
     )
 
-    ca_certs = Unicode(config=True)
+    admin_groups = Set(
+        config=True,
+        help="""
+        Allow members of selected OpenShift groups to sign in and consider them
+        as JupyterHub admins.
 
-    allowed_groups = Set(
+        If this is set and a user isn't part of one of these groups or listed in
+        `admin_users`, a user signing in will have their admin status revoked.
+        """,
+    )
+
+    ca_certs = Unicode(
         config=True,
-        help="Set of OpenShift groups that should be allowed to access the hub.",
+        help="""
+        Path to a certificate authority (CA) certificate file. Used to trust the
+        certificates from a specific CA.
+        """,
     )
 
-    admin_groups = Set(
+    # FIXME: validate_cert is defined here, but OAuthenticator also defines
+    #        validate_server_cert. If both should exist separately its too
+    #        confusing without further documentation, and if only one should
+    #        exist the one here should be deprecated in favor of the other.
+    #
```

Comment:
To be honest, I think this should be deprecated in favour of the one from the base class. It might just be a leftover from the refactor

Reply:
Ah I suspected it could be!

I have another smaller PR I plan to open if this resolves related to deprecated config, I can have a look after